server/routes/companyRoutes.js

```diff
@@ -63,6 +63,23 @@ router.get('/profile/:companyId', (req, res) => {
     })
 })
 
+router.post('/profile/:id', (req,res) => {
+  //grabs id from the url
+  const id = req.params.id
+  const body = req.body
+  body.company_id = id
+  db.addProfile(body)
+  .then(id => {
+    //returns fully fleshed object to the display company profile page, after edit is hit
+    res.json(body)
```

Comment:
You should add the `id` to this object before returning it.

Reply:
Completed